### PR TITLE
Implement Employment EscoOccupations

### PR DESCRIPTION
**Pull request by omarudolley**
Implement Employment EscoOccupations definition

### CODE REVIEW

File: DataProducts/draft/Employment/EscoOccupations.json [hunk 1]
```diff
@@ -0,0 +1,3225 @@
...
+        "required": ["occupations", "totalCount"],
+        "type": "object",
+        "properties": {
+          "occupations": {
+            "title": "Occupations",
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Occupation"
+            },
+            "description": "A list of relevant occupation classes based on ESCO standard"
```
[the remark this answers is not in the document]
Comment (hsuomi) [reply DataProducts/draft/Employment/EscoOccupations.json:3131]: The change has also been aligned in the spec and highlighted with red.

**Review by hsuomi — CHANGES_REQUESTED** [review]

File: DataProducts/draft/Employment/EscoOccupations.json [hunk 2]
```diff
@@ -0,0 +1,3225 @@
...
+              {
+                "$ref": "#/components/schemas/EscoCode"
+              }
+            ],
+            "description": "The ESCO code based on the level 5 or narrower in ESCO classification",
+            "example": "8331.1"
+          },
+          "escoJobTitle": {
+            "title": "ESCO Job Title",
+            "maxLength": 40,
```
Comment (hsuomi) [inline DataProducts/draft/Employment/EscoOccupations.json:3171]: Job Title max length must be increased to 250 characters

File: DataProducts/draft/Employment/EscoOccupations.json [hunk 3]
```diff
@@ -0,0 +1,3225 @@
...
+            "type": "string",
+            "description": "The job title based on ESCO classification",
+            "example": "bus driver"
+          },
+          "escoDescription": {
+            "title": "ESCO Job Title Description",
+            "maxLength": 250,
```
Comment (hsuomi) [inline DataProducts/draft/Employment/EscoOccupations.json:3178]: Let's remove the max length validation from ESCO description. They can occationally be rather long > 1000 characters.

**Review by hsuomi — CHANGES_REQUESTED** [review]

File: src/draft/Employment/EscoOccupations.py [hunk 4]
```diff
@@ -0,0 +1,3091 @@
...
+    ESCO_9612_2 = "9612.2"
+    ESCO_5414_1_2 = "5414.1.2"
+    ESCO_2422_1 = "2422.1"
+    ESCO_8211_5 = "8211.5"
+
+
+class Occupation(CamelCaseModel):
+    esco_code: EscoCode = Field(
+        ...,
+        title="ESCO codes",
```
Comment (joakimnordling) [inline src/draft/Employment/EscoOccupations.py:3021]:
```suggestion
        title="ESCO Code",
```

**Review by joakimnordling — CHANGES_REQUESTED** [review]

**Review by hsuomi — APPROVED** [review]

**Review by joakimnordling — APPROVED** [review]

**Pull request merged**